docker/suite/run_suite.sh

```diff
@@ -103,7 +101,6 @@ GCE_CONFIG="gce:
   region: ${GCE_REGION}
   ssh_key_path: /robotest/config/ops.pem
   ssh_pub_key_path: /robotest/config/ops_rsa.pub
-  docker_device: \"${DOCKER_DEVICE:-}\"
```

Comment:
I also removed docker device support from GCE as it was not being used in tests.